packages/cli-core/src/package-manager-controller/package_manager_controller.ts

```diff
@@ -1,11 +1,18 @@
 import { existsSync as _existsSync } from 'fs';
 import _fsp from 'fs/promises';
-import { execa as _execa } from 'execa';
+import { ExecaChildProcess, execa as _execa } from 'execa';
 import * as _path from 'path';
-import { logger } from '../logger.js';
-import { executeWithDebugLogger as _executeWithDebugLogger } from '../execute_with_logger.js';
+import { BackendLocator } from '@aws-amplify/platform-core';
+import { LogLevel } from '../printer/printer.js';
+import { printer } from '../printer.js';
+import { executeWithDebugLogger as _executeWithDebugLogger } from './execute_with_logger.js';
 import { DependencyType } from './package_manager_controller_factory.js';
 
+enum InvokableCommand {
+  DEPLOY = 'deploy',
+  DESTROY = 'destroy',
+}
```

Comment:
the CDK related stuff should remain in deployer package.

Reply:
Done 1467f8fd843c11304b13c8c17ef1414ae0a4e326